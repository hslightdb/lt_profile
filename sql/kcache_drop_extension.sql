/* Testing drop server with data */
SELECT * FROM profile.drop_server('local');
DROP EXTENSION lt_profile;
DROP EXTENSION pg_stat_kcache;
DROP EXTENSION system_stats;
DROP ROLE monitor_system_stats;
DROP EXTENSION pg_wait_sampling;
DROP EXTENSION lt_stat_statements;
DROP EXTENSION dblink;
DROP SCHEMA profile;
DROP SCHEMA dblink;
DROP SCHEMA statements;
DROP SCHEMA kcache;
