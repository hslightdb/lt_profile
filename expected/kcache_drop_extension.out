/* Testing drop server with data */
SELECT * FROM profile.drop_server('local');
 drop_server 
-------------
           1
(1 row)

DROP EXTENSION lt_profile;
DROP EXTENSION pg_stat_kcache;
DROP EXTENSION system_stats;
DROP ROLE monitor_system_stats;
DROP EXTENSION pg_wait_sampling;
DROP EXTENSION pg_stat_statements;
DROP EXTENSION dblink;
DROP SCHEMA profile;
DROP SCHEMA dblink;
DROP SCHEMA statements;
DROP SCHEMA kcache;
